praw/models/reddit/reasons.py

```diff
@@ -0,0 +1,170 @@
+"""Provide the Reason class."""
+from ...const import API_PATH
+from ...exceptions import ClientException
+from .base import RedditBase
+
+
+class Reason(RedditBase):
+    """An individual Reason object.
+
+    **Typical Attributes**
+
+    This table describes attributes that typically belong to objects of this
+    class. Since attributes are dynamically provided (see
+    :ref:`determine-available-attributes-of-an-object`), there is not a
+    guarantee that these attributes will always be present, nor is this list
+    necessarily comprehensive.
+
+    ======================= ===================================================
+    Attribute               Description
+    ======================= ===================================================
+    ``id``                  The id of the removal reason.
+    ``title``               The title of the removal reason.
+    ``message``             The message of the removal reason.
+    ======================= ===================================================
+    """
+
+    STR_FIELD = "id"
+
+    def __eq__(self, other):
+        """Return whether the other instance equals the current."""
+        if isinstance(other, str):
+            return other == str(self)
+        return (
+            isinstance(other, self.__class__)
+            and str(self) == str(other)
+            and other.subreddit == self.subreddit
+        )
+
+    def __hash__(self):
+        """Return the hash of the current instance."""
+        return (
+            hash(self.__class__.__name__)
+            ^ hash(str(self))
+            ^ hash(self.subreddit)
+        )
+
+    def __init__(self, reddit, subreddit, reason_id, _data=None):
+        """Construct an instance of the Reason object."""
+        self.id = reason_id
+        self.subreddit = subreddit
+        super(Reason, self).__init__(reddit, _data=_data)
+
+    def _fetch(self):
+        for reason in self.subreddit.reasons:
+            if reason.id == self.id:
+                self.__dict__.update(reason.__dict__)
+                self._fetched = True
+                return
+        raise ClientException(
+            "/r/{} does not have the reason {}".format(self.subreddit, self.id)
```

Comment:
Can we use just `r/{}`? I think someone recently applied the shorter formatting to this project.